presto-oracle/src/main/java/io/prestosql/plugin/oracle/OracleClient.java

```diff
@@ -270,9 +268,10 @@ public void createSchema(JdbcIdentity identity, String schemaName)
                         oracleDoubleWriteFunction(),
                         FULL_PUSHDOWN));
             case OracleTypes.NUMBER:
+                int oraclePrecision = typeHandle.getRequiredColumnSize();
```

Comment:
Maybe `fullPrecision` would be a better name? (Doesn't matter either way.)

Reply:
"actualPrecision"? "originalPrecision"?

Reply:
Anything like that works, but "oraclePrecision" doesn't capture the right semantics.

Reply:
changed to `actualPrecision`